src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtLocalAuthoritiesEndpointTest.java

```diff
@@ -0,0 +1,163 @@
+    private static final String AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG = "aylesbury-county-court-and-family-court/";
+    private static final String AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_AREAS_OF_LAW = "Money claims/";
+
+    private static final String AYLESBURY_COURT_LOCAL_AUTHORITIES_AREAS_OF_LAW_PATH = ADMIN_COURTS_ENDPOINT + AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG
+        + AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_AREAS_OF_LAW + LOCAL_AUTHORITIES_PATH;
+    //changed local authority to existing one on database.
+    private static final String TEST = "Barnet Borough Council";
+    private static final int TEST_ID = 397_243;
+
+
+    //"/{slug}/{areaOfLaw}/localAuthorities")
+
+
+    @Test
+    public void returnAllLocalAuthorities() {
+        final var response = doGetRequest(
+            ALL_LOCAL_AUTHORITIES_FULL_PATH,
+            Map.of(AUTHORIZATION, BEARER + authenticatedToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+
+        final List<LocalAuthority> localAuthorities = response.body().jsonPath().getList(".", LocalAuthority.class);
+        assertThat(localAuthorities).hasSizeGreaterThan(1);
+
+    }
+
+    @Test
+    public void shouldRequireATokenWhenGettingLocalAuthorities() {
+        final var response = doGetRequest(ALL_LOCAL_AUTHORITIES_FULL_PATH);
+        assertThat(response.statusCode()).isEqualTo(UNAUTHORIZED.value());
+    }
+
+    @Test
+    public void shouldBeForbiddenForGettingLocalAuthorities() {
+        final var response = doGetRequest(
+            ALL_LOCAL_AUTHORITIES_FULL_PATH,
+            Map.of(AUTHORIZATION, BEARER + forbiddenToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
+    }
+
+
+    @Test
+    public void returnLocalAuthoritiesForTheCourtAsPerTheAreasOfLaw() {
+        final var response = doGetRequest(
+            AYLESBURY_COURT_LOCAL_AUTHORITIES_AREAS_OF_LAW_PATH,
+            Map.of(AUTHORIZATION, BEARER + authenticatedToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+
+        final List<LocalAuthority> localAuthorities = response.body().jsonPath().getList(".", LocalAuthority.class);
+        assertThat(localAuthorities).hasSizeGreaterThan(1);
+
+    }
+
+    @Test
+    public void shouldRequireATokenWhenGettingLocalAuthoritiesForAreasOfLaw() {
+        final var response = doGetRequest(AYLESBURY_COURT_LOCAL_AUTHORITIES_AREAS_OF_LAW_PATH);
+        assertThat(response.statusCode()).isEqualTo(UNAUTHORIZED.value());
+    }
+
+    @Test
+    public void shouldBeForbiddenForGettingLocalAuthoritiesForAreasOfLaw() {
+        final var response = doGetRequest(
+            AYLESBURY_COURT_LOCAL_AUTHORITIES_AREAS_OF_LAW_PATH,
+            Map.of(AUTHORIZATION, BEARER + forbiddenToken)
+        );
+        assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
+    }
+
```

Comment:
I would add another test here to test that the standard admin user would be forbidden to access the update endpoint

Reply:
added adminShouldBeForbiddenForUpdatingLocalAuthoritiesForAreasOfLaw